.gitignore

```diff
@@ -24,9 +26,13 @@ etc/dev-frontend.yml
 .env.production.local
 go.work
 go.work.sum
+/zrok-venv
 
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
 .obsidian
+sdk/examples/pastebin/cmd/copyto/copyto
+sdk/examples/pastebin/cmd/copyto/payload
+sdk/examples/pastebin/cmd/pastefrom/pastefrom
```

Comment:
?

Reply:
I also want to know why I was thinking? These paths a) don't make sense and b) wouldn't even be useful?